app/preload.js

```diff
@@ -16,15 +16,24 @@ function navigate(url) {
 
 ipcRenderer.on('isPlaying', (event) => {
   const isPlaying = !!document.querySelector('.playing')
-  event.sender.send('isPlaying', isPlaying)
+  const icon = document.getElementsByClassName('playbackSoundBadge__avatar sc-media-image')[0].children[0].children[0].getAttribute('style')
+  var urlStart = icon.indexOf('url("') + 5;
+  var urlEnd = icon.indexOf('");') - urlStart;
+  var iconURL = icon.substr(urlStart, urlEnd)
+  event.sender.send('isPlaying', isPlaying, iconURL)
 })
 ipcRenderer.on('navigate', (_, url) => {
   navigate(url)
 })
 
 const Notification = window.Notification
-ipcRenderer.on('notification', (_, title, body) => {
-  new Notification(title, { body, silent: true })
+ipcRenderer.on('notification', (_, title, body, icon) => {
+  var options = {
+    body: body,
+    icon: icon,
```

Comment:
Please use `const` whenever possible.

Also, in ES6 if the object key is the same as the value you can omit the key in the object literal. A shorter version of this would be:

```js
const options = {
  body,
  icon,
  silent: true
}
```